h2o-test-support/src/main/java/water/TestUtil.java

```diff
@@ -1371,4 +1374,21 @@ public Vec createRandomBinaryWeightsVec(final long len, final int randomSeed) {
     return weightsVec;
   }
 
+  public static GenModel toMojo(Model model, String testName, boolean readModelMetaData) {
+    final String filename = testName + ".zip";
+    StreamingSchema ss = new StreamingSchema(model.getMojo(), filename);
+    try {
+      FileOutputStream os = new FileOutputStream(ss.getFilename());
+      ss.getStreamWriter().writeTo(os);
```

Comment:
if this line throws an exception, does `os` get closed?

Reply:
good catch, improved